include/vkgcDefs.h

```diff
@@ -834,6 +844,24 @@ class IPipelineDumper {
   /// @param [in]  nameBufSize    Size of the buffer to store pipeline name
   static void VKAPI_CALL GetPipelineName(const ComputePipelineBuildInfo *pPipelineInfo, char *pPipeName,
                                          const size_t nameBufSize);
+
+  /// Gets graphics pipeline name.
+  ///
+  /// @param [in]  pPipelineInfo   Info to build this graphics pipeline
+  /// @param [out] pPipeName       The full name of this graphics pipeline
+  /// @param [in]  nameBufSize     Size of the buffer to store pipeline name
+  /// @param hashCode64            Precalculated Hash code of pipeline
+  static void VKAPI_CALL GetPipelineName(const GraphicsPipelineBuildInfo *pPipelineInfo, char *pPipeName,
```

Comment:
I prefer changing the name for those two entrypoints:
`GetGraphicsPipelineName GetComputePipelineName`
Although we can use function overloading, it is more clear that we can use the names to make them explicit.

Reply:
these function are used in the template function. so hence  can not be changed .see
template<class PipelineBuildInfo>
bool PipelineCompiler::ReplacePipelineBinary( in the pipeline_compiler.cpp